### PR TITLE
test: weldr api module_hotfixes flag

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Adds testing for module_hotfixes on weldr API

### CODE REVIEW

File: internal/client/blueprints_test.go [hunk 1]
```diff
@@ -1006,6 +1006,35 @@ func TestBlueprintDepsolveGlobsV0(t *testing.T) {
...
+		t.Skip()
+	}
+	bp := `{
+		"name": "test-deps-blueprint-module-hotfixes-v0",
+		"description": "CheckBlueprintDepsolveModuleHotfixesV0",
+		"version": "0.0.1",
+		"packages": [{"name": "nginx", "version": "*"},
+		{"name": "nginx-module-njs", "version": "*"}]
+		"customizations": {"repositories": [{
+			"id": "nginx", "name": "nginx", "baseurls": ["https://rpmrepo.osbuild.org/v2/mirror/public/el8/el8-x86_64-nginx-20231207"], "gpgcheck"": false}
```
Comment (bcl) [inline internal/client/blueprints_test.go:1022]: You should set module_hotfixes to false to make it clear what's happening here.
Also, shouldn't you also test it with it set to true?

Comment (ezr-ondrej) [reply internal/client/blueprints_test.go]: oh, sorry it was not really ready, I supposed to have drafted it :)
Now it's what I believe we should add, but it still doesn't work locally, I think it uses a my local dnfjson maybe? :)

Comment (ezr-ondrej) [reply internal/client/blueprints_test.go]: Well, I've pushed next rebase, let see what happens :shrug:
If it stays red, I'm out of ideas :thinking:

**Comment by ezr-ondrej** [comment]
It's still very red :( xD

**Pull request closed without merging**